src/main/scala/io/github/sentenza/hacktoberfest18/MenuIO.scala

```diff
@@ -39,23 +40,38 @@ object MenuIO {
      * This function should be called at the very beginning of the Main execution
      * to fetch the disclaimer message and the project Logo to be printed out
      */
-    private def printDisclaimer() { println(heading + gplDisclaimer) }
+    def printDisclaimer() { println(heading + gplDisclaimer) }
 
+    private val noOp = () => ()
+
+    case class MenuEntry(selector: Int, display: String, code: () => Unit)
```

Comment:
That's a very nice one. I like the use of an a anonymous function as part of the case class. 👍

Reply:
thanks, I have doubts if it will fit all use cases... we'll see